.github/workflows/build.yaml

```diff
@@ -12,7 +12,7 @@ jobs:
     runs-on: self-hosted
     timeout-minutes: 20
     container:
-      image: docker.io/mayadata/ms-buildenv:nix
+      image: docker.io/teamrebuild/ms-buildenv:spdk-changes
```

Comment:
You are creating a lot of work for yourselves this way. I’m fine with it, just saying that merging all of this back is not going to be trivial.

Reply:
This is meant to be temporary until we update the SPDK version. It was the best way we could think of to get the build to pass.